### ISSUE TITLE
1.5 Getting Started - Installing Git

### ISSUE THREAD

**Issue by Maurice333** [git/git-scm.com#558]
TL;DR [This book](http://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) frustrated the hell out of me by not providing what I consider enough information.

Under the Installing on Windows the author tells us that another easy way to install git on Windows is by following the link to [http://windows.github.com](http://windows.github.com). I found that it had nothing to do with the examples that followed in chapter 1.6.

I figured I was just inept and looked in the videos section for guidance- which did nothing for me. The video I hoped would help me "Get Going with Git" skipped the entire install process. It tells you "where" to get Git for [your OS] but that's it. The next thing you see is this:

 ![Imgur](http://i.imgur.com/570BYP7.png)

[This YouTube video](https://www.youtube.com/watch?v=_tN0T7jYn0A) I found was more informative and in-line with the examples that followed in chapter 1.6. It at least shows someone like me who has no idea what they're doing the little steps in between instead of saying "get Git here, okay let's start!"

I saw the opportunity to make suggestions at the bottom of the web page and voilà!

**Comment by sxlijin** [comment]
Is this still an issue? I realize at the time of this issue the Git for Windows project was still relatively young, but I think it's decently mature now; also, I can't find any reference to windows.github.io in the linked page.

This is also actually about the Pro Git book - I think - so you probably want to reopen this issue over at [that repository](https://github.com/progit/progit2).

**Comment by peff** [comment]
Yes, I think this has matured a lot outside the site. Will close for now; re-open if it remains an issue.
